aws/tf/sra.tf

```diff
@@ -5,66 +5,34 @@ module "SRA" {
-  read_only_external_location_admin          = null  // Admin for the external location.
-
-  enable_cluster_boolean       = false // Set to true to create a default Databricks clusters.
-  enable_admin_configs_boolean = false // Set to true to enable optional admin configurations.
-  enable_logging_boolean       = false // Set to true to enable log delivery and creation of related assets (e.g. S3 bucket and IAM role)
-
-  enable_restrictive_root_bucket_boolean      = false // Set to true to enable a restrictive root bucket policy, this is subject to change and may cause unexpected issues in the event of a change.
-  enable_restrictive_s3_endpoint_boolean      = false // Set to true to enable a restrictive S3 endpoint policy, this is subject to change and may cause unexpected issues in the event of a change.
-  enable_restrictive_sts_endpoint_boolean     = false // Set to true to enable a restrictive STS endpoint policy, this is subject to change and may cause unexpected issues in the event of a change.
-  enable_restrictive_kinesis_endpoint_boolean = false // Set to true to enable a restrictive Kinesis endpoint policy, this is subject to change and may cause unexpected issues in the event of a change.
-
-  enable_ip_boolean = false                                   // Set to true to enable IP access list.
-  ip_addresses      = ["X.X.X.X", "X.X.X.X/XX", "X.X.X.X/XX"] // Specify IP addresses for access.
-
-  enable_system_tables_schema_boolean = false // Set to true to enable system table schemas
-
-  enable_sat_boolean        = false // Set to true to enable Security Analysis Tool. https://github.com/databricks-industry-solutions/security-analysis-tool
-  enable_audit_log_alerting = false // Set to true to create 40+ queries for audit log alerting based on user activity. https://github.com/andyweaves/system-tables-audit-logs
+  region_bucket_name    = var.region_bucket_name[var.region]
+  admin_user            = var.admin_user
+  resource_prefix       = var.resource_prefix
+
+  // REQUIRED:
+  network_configuration = "isolated" // Network (custom or isolated), see README.md for more information.
+  metastore_exists      = false      // If a regional metastore exists set to true.
+
+  // REQUIRED IF USING ISOLATED NETWORK:
+  vpc_cidr_range           = "10.0.0.0/18" // Please re-define the subsequent subnet ranges if the VPC CIDR range is updated.
+  private_subnets_cidr     = ["10.0.0.0/22", "10.0.4.0/22", "10.0.8.0/22"]
+  privatelink_subnets_cidr = ["10.0.28.0/26", "10.0.28.64/26", "10.0.28.128/26"]
+  availability_zones       = [data.aws_availability_zones.available.names[0], data.aws_availability_zones.available.names[1], data.aws_availability_zones.available.names[2]]
+  sg_egress_ports          = [443, 3306, 6666, 8443, 8444, 8445, 8446, 8447, 8448, 8449, 8450, 8451]
+
+  // REQUIRED IF USING NON-ROOT ACCOUNT CMK ADMIN:
+  # cmk_admin_arn    = null       // CMK admin ARN, defaults to the AWS account root.
```

Comment:
I think we should include examples instead of "null" so that customers know what the resource ID looks like, often they get confused about what ID to copy

Reply:
fair point, let me add this